RF01-TC-037

danielrpg opened the pull request:
UI > Registro de Usuario > Verificar cabecera de lista de Usuario (UserName, Password, Active, Accion) deben ser mostrados correctamente.

danielrpg commented:
![image](https://user-images.githubusercontent.com/1975854/41692917-cd067bea-74d0-11e8-9c7c-3c247124a43e.png)

danielrpg commented:
![image](https://user-images.githubusercontent.com/1975854/41692936-e79240ca-74d0-11e8-8c06-70eb59b4f616.png)

danielrpg committed:
RF01-TC-037

UI > Registro de Usuario > Verificar cabecera de lista de Usuario (UserName, Password, Active, Accion) deben ser mostrados correctamente.

marcelogaray commented on src/main/java/org/umssdiplo/automationv01/core/managepage/Header/Header.java:
```diff
@@ -0,0 +1,33 @@
+        }else{
+            System.out.println("No existe el menu personal");
+        }
+
```
remove blank line

marcelogaray commented on src/main/java/org/umssdiplo/automationv01/core/managepage/Header/Header.java:
```diff
@@ -0,0 +1,33 @@
+        if(CommonEvents.isVisible(personalMenu)){
+            CommonEvents.clickButton(personalMenu);
+        }else{
+            System.out.println("No existe el menu personal");
```
avoid use SOUTP

try to use LOGGERS

marcelogaray commented on src/main/java/org/umssdiplo/automationv01/core/managepage/Header/Header.java:
```diff
@@ -0,0 +1,33 @@
+    public void selectSubMenuUsuario() {
+        CommonEvents.clickButton(usuarioSubMenu);
+    }
+
```
remove blank line

marcelogaray commented on src/main/java/org/umssdiplo/automationv01/core/managepage/Header/Header.java:
```diff
@@ -0,0 +1,33 @@
+    }
+
+    public void selectSubMenuUsuario() {
+        CommonEvents.clickButton(usuarioSubMenu);
```
which is the page object that you expected in this point?

marcelogaray commented on src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java:
```diff
@@ -17,6 +18,16 @@ public void ssiDPaginaDeInicioDeSesionEsCargada() throws Throwable {
+
+    @Given("^seleccionar el menu de persona$")
+    public void seleccionarElMenuDePersona() throws Throwable {
+        header.selectPersonalMenu();
```
user page objects

marcelogaray reviewed (changes requested):
review observations

danielrpg committed:
RF01-TC-037

UI > Registro de Usuario > Verificar cabecera de lista de Usuario (UserName, Password, Active, Accion) deben ser mostrados correctamente.

danielrpg committed:
Merged Changes

danielrpg committed:
RF01-TC-037

UI > Registro de Usuario > Verificar cabecera de lista de Usuario (UserName, Password, Active, Accion) deben ser mostrados correctamente.

marcelogaray commented on src/main/java/org/umssdiplo/automationv01/core/managepage/Usuario/ListUser.java:
```diff
@@ -0,0 +1,23 @@
+        CommonEvents.isVisible(btnAgregarUsuario);
+    }
+
+    public boolean verifyListUser() {
```
it will be better if you use 'isUserListVisible' instead of 'verifyListUser'.

marcelogaray commented on src/test/resources/features/users/listUsers.feature:
```diff
@@ -0,0 +1,12 @@
+    Given 'SSI-D' pagina de inicio de sesion es cargada
+    And Ingresar los credenciales validos en la pagina 'Login' para ingresar al sistema
+
+  Scenario:
```
you need to fill your test case title.

marcelogaray commented on src/test/resources/features/users/listUsers.feature:
```diff
@@ -0,0 +1,12 @@
+
+  Scenario:
+    Given Menu principal estee cargado
+    And Menu 'Personal' estee seleccionado
```
And seleccionar menu 'Personal' en la pagina 'Menu principal'

marcelogaray commented on src/test/resources/features/users/listUsers.feature:
```diff
@@ -0,0 +1,12 @@
+  Scenario:
+    Given Menu principal estee cargado
+    And Menu 'Personal' estee seleccionado
+    And Seleccionar SubMenu 'Usuario'
```
And seleccionar submenu 'Usuario' en menu 'Personal'

marcelogaray commented on src/test/resources/features/users/listUsers.feature:
```diff
@@ -0,0 +1,12 @@
+    Given Menu principal estee cargado
+    And Menu 'Personal' estee seleccionado
+    And Seleccionar SubMenu 'Usuario'
+    Then Validar 'Lista de Usuarios'
```
Then validar que la 'Lista de usuarios' este visible.

marcelogaray reviewed (changes requested):
review observations

danielrpg committed:
RF01-TC-037

Applying changes observed by reviewer
